### PR TITLE
Card: Updating examples

### PR DESCRIPTION
**Pull request by khmakoto**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Updating `Card` examples to add a narrow card (fixed width) with its contents, clean some imports and remove the use of `mergeStyleSets` and `gap`.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8590)

### CODE REVIEW

File: packages/react-cards/src/components/Card/examples/Card.Compact.Example.tsx [hunk 1]
```diff
@@ -2,59 +2,79 @@
 import * as React from 'react';
 import { Text } from '@uifabric/experiments';
 import { Icon, Image, Stack } from 'office-ui-fabric-react';
-import { mergeStyleSets, FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
 import { Card } from '../Card';
```
Comment (cliffkoh) [inline packages/react-cards/src/components/Card/examples/Card.Compact.Example.tsx]: All examples should do "absolute" package import so it is easier to codepen

File: packages/react-cards/src/components/Card/examples/Card.Basic.Example.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import * as React from 'react';
 import { Persona, Text } from '@uifabric/experiments';
 import { Icon, Image, Stack } from 'office-ui-fabric-react';
-import { mergeStyleSets, FontWeights } from 'office-ui-fabric-react/lib/Styling';
+import { FontWeights } from 'office-ui-fabric-react/lib/Styling';
 import { Card } from '../Card';
```
Comment (cliffkoh) [inline packages/react-cards/src/components/Card/examples/Card.Basic.Example.tsx]: change to package import

File: packages/react-cards/src/components/Card/examples/Card.Compact.Example.tsx [hunk 3]
```diff
@@ -2,59 +2,79 @@
 import * as React from 'react';
 import { Text } from '@uifabric/experiments';
```
Comment (ecraig12345) [inline packages/react-cards/src/components/Card/examples/Card.Compact.Example.tsx]: Isn't Text in OUFR now?

Comment (khmakoto) [reply packages/react-cards/src/components/Card/examples/Card.Compact.Example.tsx]: Yes, let me update that as well. These examples were first created before that promotion. Thanks for catching this! 😄

**Pull request merged**